### PR TITLE
maxOfVector done

### PR DESCRIPTION
**Pull request by BarTes8**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: max-of-vector
🏅 Granted 5 XP for @BarTes8

**Comment by ziobron** [comment]
⏱ 3 XP extra for @BarTes8 for on-time delivery

File: homework/max-of-vector/maxOfVector.hpp [hunk 1]
```diff
@@ -1,8 +1,9 @@
 #pragma once
 #include <vector>
 #include <limits>
```
Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:3]: remove if not used

File: homework/max-of-vector/maxOfVector.hpp [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
+#include <algorithm>
 
 int maxOfVector(const std::vector<int>& vec) {
-    // TODO: Implement me :)
-    return {};
+    auto result = std::max_element(vec.begin(), vec.end());
+    return *result;
```
Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:8]: just return *std::max_element(vec.begin(), vec.end());

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**